backend-server/src/main/java/org/dice/ida/logging/AspectLogger.java

```diff
@@ -0,0 +1,107 @@
+package org.dice.ida.logging;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.dice.ida.constant.IDAConst;
+import org.dice.ida.model.ChatMessageResponse;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Component
+@Aspect
+@EnableAspectJAutoProxy
+public class AspectLogger {
+
+	@Autowired
+	private Logger log;
+
+	@Autowired
+	private HttpServletRequest request;
+
+	@Autowired
+	private ChatMessageResponse response;
+
+	/**
+	 * Creates the point cut for the MessageRestController
+	 *
+	 */
+	@Pointcut("execution(* org.dice.ida.controller.*.*(..))")
+	public void controller() {
+		//PointCut for MessageRestController
+	}
+
+	/**
+	 * Method to log the Request to Message Rest Controller
+	 * @param joinPoint
+	 */
+	@Before("controller()")
+	public void logBeforeMethod(JoinPoint joinPoint) {
+		StringBuffer logMessage = new StringBuffer();
+		logMessage.append("[REQUEST] - ");
+		commonMsgBody(joinPoint, logMessage);
+		log.info(logMessage.toString());
+	}
+
+	/**
+	 * Method to log the exceptions
+	 * @param joinPoint
+	 * @param exception
+	 */
+	@AfterThrowing(pointcut = "execution(* org.dice.ida.*.*.*.*(..))", throwing = "exception")
```

Comment:
This looks like a very specific pattern and is limited by the depth. Please use something like `"execution(* org.dice.ida..*(..))"` (Ref: https://www.baeldung.com/spring-aop-pointcut-tutorial#4-args)